### PR TITLE
Add wilmersdorf, tron legacy and yay-evil distro

### PR DESCRIPTION
**Pull request by ianpan870102**
Links to the repos:
- Wilmersdorf: https://github.com/ianpan870102/wilmersdorf-emacs-theme
- Tron Legacy: https://github.com/ianpan870102/tron-legacy-emacs-theme
- Yay-Evil Distro: https://github.com/ianpan870102/.emacs.d

### CODE REVIEW

**Comment by ianpan870102** [comment]
Wilmersdorf and Tron Legacy have both proven popularity on forums like Reddit's r/emacs and are featured on GNU Emacs Theme website.

The Yay-Evil Distro that I currently maintain contains a comprehensive starter kit experience with good defaults, documented in literate programming Org style.

File: README.org [hunk 1]
```diff
@@ -836,6 +836,8 @@ For additional git related emacs packages to use or to get inspiration from, tak
    - [[https://framagit.org/steckerhalter/grandshell-theme][Grandshell-theme]] - /(dark)/ *[256color]* Theme with intensive colors.
    - [[https://github.com/kunalb/poet][Poet]] - /(light/dark)/ A lightweight theme that plays well with prose and org-mode.
    - [[https://github.com/ogdenwebb/emacs-kaolin-themes][Kaolin-themes]] - (theme package) Set of eye pleasing themes for GNU Emacs. Supports both GUI and terminal.
+   - [[https://github.com/ianpan870102/wilmersdorf-emacs-theme][Wilmersdorf-theme]] - /(dark)/ Emacs theme with dark subtle syntax highlighting
```
Comment (agsdot) [inline README.org]: Please end the description with 

syntax highlighting.

File: README.org [hunk 2]
```diff
@@ -836,6 +836,8 @@ For additional git related emacs packages to use or to get inspiration from, tak
...
+   - [[https://github.com/ianpan870102/tron-legacy-emacs-theme][Tron-Legacy-Theme]] - /(dark)/ Custom theme inspired by Tron: Legacy
```
Comment (agsdot) [inline README.org]: Once again. Add a period to finish the description.

Tron: Legacy.

**Comment by ianpan870102** [comment]
Thanks for your time, @agsdot. I've fixed the typo and squashed my latest commit to the previous one to keep the commit history clean. Let me know if it's all good!

Screenshot 2019-07-26 at 10 06 04 AM

File: README.org (hunk 2 above)
Comment (ianpan870102) [reply README.org]: Fixed! Please refer to my latest comment.

File: README.org (hunk 1 above)
Comment (ianpan870102) [reply README.org]: Fixed! Please refer to my latest comment.

**Comment by agsdot** [comment]
Thanks @ianpan870102!

**Pull request merged**